### PR TITLE
Fix from GitHub comments

### PR DESCRIPTION
**Pull request by vincentchalamon**
- [x] Fix comments from https://github.com/dunglas/DunglasApiBundle/pull/242

### CODE REVIEW

**Comment by teohhanhui** [comment]
Take your time. And thanks!

File: Tests/Doctrine/Orm/SearchFilterTest.php [hunk 1]
```diff
@@ -1,183 +0,0 @@
...
-use phpmock\phpunit\PHPMock;
-use Symfony\Bridge\Doctrine\Test\DoctrineTestHelper;
-use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\RequestStack;
-
-/**
- * @author Julien Deniau <julien.deniau@mapado.com>
- */
-class SearchFilterTest extends KernelTestCase
```
Comment (teohhanhui) [inline Tests/Doctrine/Orm/SearchFilterTest.php:28]: Just curious what happened to this file...

Comment (vincentchalamon) [inline Tests/Doctrine/Orm/SearchFilterTest.php:28]: Duplicate from `Tests/Doctrine/Orm/Filter/SearchFilterTest.php`

**Comment by vincentchalamon** [comment]
@teohhanhui Good, classes renamed !

**Comment by teohhanhui** [comment]
Thanks. I also noticed that we should be doing `$request->getContent()` instead of `file_get_contents('php://input')`

I'll fix that.

**Comment by vincentchalamon** [comment]
OK, I let you fix it

**Comment by vincentchalamon** [comment]
@teohhanhui `$request->getContent()` fixed

**Pull request merged**